`update-rust-toolchain` workflow is broken

This appears to be an upsteam bug. I'm seeing errors like this:

```
Run a-kenji/update-rust-toolchain@v1
Run bash $GITHUB_ACTION_PATH/update-toolchain.sh
RUST_TOOLCHAIN_FILE ./rust-toolchain.toml
MINOR_DELTA 0
UPDATE_PATCH true
UPDATE_MINOR true
RUST_TOOLCHAIN_VERSION 1.93.0
CURRENT_RUST_TOOLCHAIN_VERSION 1.93.0
RUST_TOOLCHAIN_VERSION_SEMVER 1 93 0
sed: -e expression #1, char [23](https://github.com/EnergySystemsModellingLab/MUSE2/actions/runs/22400641949/job/64845991343#step:3:25): unterminated `s' command
Error: Process completed with exit code 1.
```

https://github.com/EnergySystemsModellingLab/MUSE2/actions/runs/22400641949/job/64845991343

--- a/.github/workflows/update-rust-toolchain.yml
+++ b/.github/workflows/update-rust-toolchain.yml
@@ -14,7 +14,7 @@ jobs:
         with:
           persist-credentials: false
       - name: update rust toolchain
-        uses: a-kenji/update-rust-toolchain@v1
+        uses: a-kenji/update-rust-toolchain@v1.1
         with:
           toolchain-path: ./rust-toolchain.toml
           token: ${{ secrets.GH_TOKEN_FOR_UPDATES }}

